src/common/span.rs

```diff
@@ -59,9 +66,7 @@ impl Span {
     /// but false there is a total tie
     /// or otherwise.
     pub fn later_than(&self, other: &Span) -> bool {
-        self.offset > other.offset
-           || (self.offset == other.offset
-              && self.end() > other.end())
```

Comment:
I think this is more readable, but it should probably be broken up into a couple expressions.